src/library/scala/collection/immutable/Seq.scala

```diff
@@ -36,6 +37,20 @@ trait Seq[+A] extends Iterable[A]
   override def toSeq: Seq[A] = this
   override def seq: Seq[A] = this
   protected[this] override def parCombiner = ParSeq.newCombiner[A] // if `immutable.SeqLike` gets introduced, please move this there!
+  override def sorted[B >: A](implicit ord: Ordering[B]): this.type = {
```

Comment:
Not really sure it this is the correct place to insert the override

There isn't an `immutable.SeqLike` which would be the ideal I think